src/tool/hpcrun/cct/cct.h

```diff
@@ -102,7 +102,6 @@
 #define ADDR(L)      (cct_addr_t) NON_LUSH_ADDR_INI(L)
 #define ADDR2_I(id, ip) NON_LUSH_ADDR_INI(id, ip)
 #define ADDR2(id, ip) (cct_addr_t) ADDR2_I(id, ip)
-#define HPCRUN_DUMMY_NODE 65534
```

Comment:
Seemingly unrelated change, remove or factor out.

(Rest of file): Inconsequential changes. Remove.